src/components/select/select-list/select-list.component.js

```diff
@@ -303,7 +359,30 @@ const SelectList = React.forwardRef(
         window.removeEventListener(keyboardEvent, handleGlobalKeydown);
         listElement.removeEventListener("scroll", handleListScroll);
       };
-    }, [handleGlobalKeydown, handleListScroll]);
+    }, [handleGlobalKeydown, handleListScroll, listContainerRef]);
+
+    // mimics virtualiser.scrollToIndex - but this doesn't work directly in the multicolumn case due
+    // to the extra space taken by the table header. So in that case we work out the offset ourselves.
+    const scrollToIndex = useCallback(
+      (targetIndex) => {
+        const scrollOptions = {
+          smoothScroll: false,
+          align: "end",
+        };
+        if (multiColumn && targetIndex > -1) {
+          // Note: virtualizer.measurementsCache isn't actually documented but isn't marked as private either,
+          // and does actually work at least for now. Unfortunately there's no way with the documented public
+          // API to get this - we can't use virtualizer.getVirtualItems() because the target child may not
+          // be currentlly rendered.
+          const { measurementsCache } = virtualizer;
```

Comment:
I think this is risky, especially considering that you're using `@tanstack/react-virtual` in beta version

Reply:
Inclined to agree with Maciek especially if we use a `beta` version